spec/controllers/api/comments_controller_spec.rb

```diff
@@ -89,18 +90,28 @@
   context 'when debate is closed' do
     let(:debate) { create(:debate, :closed_debate) }
     let(:auth_token) { debate.auth_tokens.create }
+    let!(:user) { create(:user, auth_token: auth_token, first_name: 'John', last_name: 'Doe') }
```

Comment:
can you use `let` instead of `let!` here?
is `first_name` and/or `last_name` needed?